src/Mvc/Mvc.ViewFeatures/src/InputType.cs

```diff
@@ -3,12 +3,34 @@
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures
 {
+    /// <summary>
+    /// An enum for input types.
```

Comment:
```suggestion
    /// Specifies constants which define an input type.
```

Per the styleguide [here](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Enum).